src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -3,18 +3,33 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Drawing;
 using System.Linq;
 using System.Reflection;
+using Res = Dynamo.Wpf.Properties.Resources;
 
 namespace Dynamo.ViewModels
 {
+    /// <summary>
+    /// The next enum will contain the posible values for Scaling (Visual Settings -> Geometry Scaling section)
+    /// </summary>
+    public enum GeometryScaleSize
+    {
+        Small,
+        Medium,
+        Large,
+        ExtraLarge
+    }
     public class PreferencesViewModel : ViewModelBase
     {
 
-        private ObservableCollection<string> _languagesList;
-        private ObservableCollection<string> _fontSizeList;
-        private ObservableCollection<string> _numberFormatList;
-        private ObservableCollection<string> _pythonEngineList;
+        private ObservableCollection<string> languagesList;
```

Comment:
We may need a #region block for easily folding these properties

Reply:
@QilongTang A new region was inserted for private properties, see commit: https://github.com/DynamoDS/Dynamo/pull/11636/commits/0fb11cde9886451ae02192f52dfc92552696ce67